rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -194,8 +145,16 @@ private void init(String url, String login, String password, boolean insecure) t
         this.url = url;
         this.login = login;
         this.password = password;
+        this.initialized = true;
```

Comment:
Should `this.initialized` value not be set after `renewSession` to avoid inconsistencies (i.e. to be the last action performed in the `init` method)?